lib/heroku/command/pg_backups.rb

```diff
@@ -628,7 +630,7 @@ def hpg_app_client(app_name)
   end
 
   def parse_schedule_time(time_str)
-    hour, tz = time_str.match(/([0-2][0-9]):00 (.*)/) && [ $1, $2 ]
+    hour, tz = time_str.match(/([0-2][0-9]):00 ?(.*)/) && [ $1, $2 ]
```

Comment:
I also had to make the space here optional because our argument parsing in the helper method seems kinda busted. I'm open to other alternatives if anyone has suggestions.